scripts/dts/gen_defines.py

```diff
@@ -611,6 +614,7 @@ def out(ident, val, aliases=()):
             # For the configuration file, the value is just repeated for all
             # the aliases
             print("DT_{}={}".format(alias, val), file=conf_file)
+    return primary_ident
```

Comment:
Could put a blank line before this line maybe, since it isn't part of the alias generation.